src/ddmd/backend/gloop.c

```diff
@@ -794,9 +795,31 @@ void loopopt()
     /* one first, thus moving LIs out as far as possible.       */
 
     doflow = TRUE;                      /* do flow analysis             */
+
+    if (go.mfoptim & MFtime)
+    {
+        cmes("Starting loop unrolling\n");
+        for (loop *l = startloop; l; l = ln)
```

Comment:
In general you're quite right, and I considered it. But it's used consistently throughout `gloop.c`, its meaning is clear from the context, and being typed as a pointer means the compiler won't let you conflate it with `1`.